tests/queries/0_stateless/02310_generate_multi_columns_with_uuid_v7.sql

```diff
@@ -0,0 +1,17 @@
+SELECT generateUUIDv7(1) = generateUUIDv7(2);
```

Comment:
Single calls to `generateUUIDv7()` will be hard to test (as the output is non-deterministic) but we can at least check that the returned value has the right length (128 bits) and the right bits set for version and variant fields.